### DISCUSSION TITLE
Nextjs + inbuilt i18n + Vercel = 502 NO_RESPONSE_FROM_FUNCTION

### DISCUSSION THREAD

**Discussion by robinsingh-bw** [vercel/vercel#5786] [Help] [answered]
Hi,

We are using i18n on nextjs 10.0.3 - there are no additional plugins for the i18n we simply leverage the inbuilt i18n support provided by nextjs out of the box.

When building on local (in prod mode) everything works fine but when we deploy to vercel this particular file results in a 502:

xyz.vercel.app/_next/data/WMyPlzgs_8L2G_E_PXKXi/en/.json

This file returns the correct json on local though. My guess is that files starting with dot may not be supported on vercel? Seems like an odd issue as others using i18n would also be affected but I haven't found any such issues online.

We tried disabling i18n and that resulted in the data being fetched from this file instead without any problems:
xyz.vercel.app/_next/data/WMyP...sadasd/index.json

Basically issue is only with /.json.

Any help would be appreciated!

Thanks

**Comment by robinsingh-bw** [comment, marked as answer]
This issue was due to setting trailingSlashes to true, setting it to false fixes it but this is obviously a bug.
